ivadomed/random.py

```diff
@@ -0,0 +1,8 @@
+import torch
+import random
+import numpy as np
+
+def set_seed(seed=6, rank=0):
```

Comment:
please add docstrings.

Reply:
pls indicate what the function does

Reply:
I also recommend adding type hinting, i.e. `(seed: int = 6, rank: int = 0)`